CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Include `braintree-core`, `three-d-secure`, `paypal`, `venmo`, `google-pay`, `card`, `data-collector`, and `union-pay` as api
+* Add `DropInClient`
+* Add `DropInResultCallback`
+* Add `FetchMostRecentPaymentMethodCallback`
+* Add `VaultedPaymentMethodSelectedListener`
+* Bump `card-form` to `5.2.0`
+* Breaking Changes
+  * Make `AddPaymentUpdateListener` package-private
+  * Make `AnimationFinishedListener` package-private
+  * Make `BaseActivity` package-private
+  * Make `SupportedPaymentMethodsAdapter` package-private
+  * Make `VaultedPaymentMethodsAdapter` package-private
+  * Make `VaultManagerPaymentMethodsAdapter` package-private
+  * Bump braintree_android module dependency versions to `4.4.1`
+  * Remove `ConfigurationListener`, `AddPaymentUpdateListener`, `PaymentMethodNonceCreatedListener`, `BraintreeErrorListener`, `BraintreeCancelListener`, and `UnionPayListener` from `AddCardActivity`
+  * Remove `ConfigurationListener`, `PaymentMethodNonceCreatedListener`, `PaymentMethodNonceUpdatedListener`, `PaymentMethodSelectedListener`, `BraintreeErrorListener`, `BraintreeCancelListener` from `DropInActivity`
+  * Remove `PaymentMethodNonceDeletedListener`, `BraintreeErrorListener` from `VaultManagerActivity`
+  * Rename `PaymentMethodType` to `DropInPaymentMethodType`
+  * Remove `DropInResultListener`
+  * Remove `DropInResult#fetchDropInResult()` 
+  * Bump braintree_android dependency versions to 4.1.0
+  * Make `DropInResult` constructor package-private
+  * Replace `AddCardActivity` with `AddCardFragment` and `CardDetailsFragment`
+  * Remove `AddCardView`
+  * Remove `EditCardView`
+  * Change `DropInPaymentMethodType#getCardTypes()` parameter from `Set` to `List`
+  * Replace `VaultManagerActivity` with `VaultManagerFragment`
+  * Remove `clientToken`, `tokenizationKey`, and `authorization` from `DropInRequest` in favor of instantiating `DropInClient` with authorization
+  * Remove deprecated `amount` parameter from `DropInRequest`
+  * Remove `intent` from `DropInRequest`
+  * Remove builder pattern from `DropInRequest`
+  * Update `DropInRequest` getter and setter methods to follow consistent naming for Kotlin synthesized properties
+  * Replace `DropInActivity.EXTRA_ERROR` with `DropInResult.EXTRA_ERROR`
+
+**Note:** The credentials for integrating with 3DS have changed. If you are using 3DS please update the credentials in your app-level `build.gradle`
```

Comment:
This is already in the README since it's actually a required change for all versions. Should we remove this CHANGELOG entry since it's alreayd in 5.2.2 or is it worth duplicating?

Reply:
Good call yeah we have something in the MIGRATION_GUIDE we could add to the README.